core/src/main/scala/org/http4s/internal/package.scala

```diff
@@ -10,4 +10,14 @@ package object internal {
   private[http4s] def unsafeRunAsync[F[_], A](fa: F[A])(
       f: Either[Throwable, A] => IO[Unit])(implicit F: Effect[F], ec: ExecutionContext): Unit =
     F.runAsync(Async.shift(ec) *> fa)(f).unsafeRunSync
+
+  private[http4s] def blocking[F[_], A](fa: F[A], blockingExecutionContext: ExecutionContext)(
+      implicit F: Async[F],
+      timer: Timer[F]): F[A] =
+    for {
+      _ <- Async.shift[F](blockingExecutionContext)
+      att <- fa.attempt
+      _ <- timer.shift
+      fa0 <- F.fromEither(att)
+    } yield fa0
```

Comment:
A little different from the impl in 0.18.x.  Should look a lot like the linebacker for the corresponding version. /cc @ChristopherDavenport

Reply:
Why is this not using bracket? Take a look at the current impl in linebacker.